src/components/templates/PartyMap/components/RoomModal/RoomModal.tsx

```diff
@@ -26,15 +24,17 @@ export interface RoomModalProps {
   show: boolean;
   venue?: AnyVenue;
   room?: Room;
+  venueEvents?: WithVenueId<WithId<VenueEvent>>[];
 }
 
 export const RoomModal: React.FC<RoomModalProps> = ({
   onHide,
   room,
   show,
   venue,
+  venueEvents,
 }) => {
-  if (!venue || !room) return null;
+  if (!venue || !room || !venueEvents) return null;
```

Comment:
I don't think we should have this check. Especially, when we check for that down in the subtree

Reply:
Removed in https://github.com/sparkletown/sparkle/pull/1538/commits/05e69b6ea9cffce574fb944d32462f8f82570d45